### PR TITLE
remove some things that do not need to be

### PR DESCRIPTION
**Pull request by matthiaskrgr**

### CODE REVIEW

File: compiler/rustc_trait_selection/src/traits/error_reporting/type_err_ctxt_ext.rs [hunk 1]
```diff
@@ -498,10 +498,6 @@ impl<'tcx> TypeErrCtxt<'_, 'tcx> {
                                 )
                             })
                             .unwrap_or_default();
-                        let file_note = file.as_ref().map(|file| format!(
-                            "the full trait has been written to '{}'",
-                            file.display(),
-                        ));
```
Comment (jieyouxu) [inline compiler/rustc_trait_selection/src/traits/error_reporting/type_err_ctxt_ext.rs:504]: Hmm, do we lose a diagnostic for where the trait has been written to if we remove this?

Comment (matthiaskrgr) [reply compiler/rustc_trait_selection/src/traits/error_reporting/type_err_ctxt_ext.rs:504]: aaaah I'll double check.
It looks like we don't have any tests here actually.

File: compiler/rustc_trait_selection/src/traits/error_reporting/type_err_ctxt_ext.rs [hunk 2]
```diff
@@ -625,7 +621,6 @@ impl<'tcx> TypeErrCtxt<'_, 'tcx> {
                             return err.emit();
                         }
 
-                        file_note.map(|note| err.note(note));
```
Comment (jieyouxu) [inline compiler/rustc_trait_selection/src/traits/error_reporting/type_err_ctxt_ext.rs:628]: Also here, I think we might lose a file note here

**Pull request merged**